cmd/dataset_tokenizer/dataset_tokenizer.go

```diff
@@ -495,8 +523,8 @@ func ReadTexts(
 
 	// We pre-emptively do the work to set up the buffers for the next files,
 	// while the prior file is being consumed.
-	runeReaders := make(chan namedRuneReader, 64)
-	paths := make(chan PathInfo, 64)
+	runeReaders := make(chan namedRuneReader, 256)
+	paths := make(chan PathInfo, 256)
```

Comment:
I'm curious why what is the reasoning behind needing to expand the channel sizes here

Reply:
Because against VAST filesystem, we need to prefetch as many files as possible and fill a pipeline.